### DISCUSSION TITLE
Insta 57008000 Cover not supported (router) or (unknown)

### DISCUSSION THREAD

**Discussion by Stefan-Mav** [Koenkk/zigbee2mqtt#12431] [Devices]
I have an Insta 57008000 Cover, i get an connection, but there is no chance to get it work.

Device.yaml:
`'0x5c0272fffebd42b5':
  friendly_name: 'Jalousie Wintergarten'
  invert_cover: false`

Later i get this in config:
Zigbee2MQTT:warn  2022-05-10 00:46:45: Received message from unsupported device with Zigbee model 'Generic UP Device' and manufacturer name 'Insta GmbH'

The firmware from 57008000 is 0.0.49.1
zigbee2mqtt is 1.18.1-1

Anyone an idea why its unsupported?
Stefan

**Comment by Koenkk** [comment]
You are using an old z2m version.

If you are still on 1.18.1 check first point of https://github.com/Koenkk/zigbee2mqtt/releases/tag/1.19.0

**Comment by Stefan-Mav** [comment]
Thanks first,

i found this:
https://github.com/zigbee2mqtt/hassio-zigbee2mqtt

But now i have trouble to get it working.
`[s6-init] making user provided files available at /var/run/s6/etc...exited 0.
[s6-init] ensuring user provided files have correct perms...exited 0.
[fix-attrs.d] applying ownership & permissions fixes...
[fix-attrs.d] done.
[cont-init.d] executing container initialization scripts...
[cont-init.d] socat.sh: executing... 
[23:10:06] INFO: Socat not enabled, marking service as down
[cont-init.d] socat.sh: exited 0.
[cont-init.d] zigbee2mqtt.sh: executing... 
[cont-init.d] zigbee2mqtt.sh: exited 0.
[cont-init.d] done.
[services.d] starting services
[services.d] done.
[23:10:08] INFO: Handing over control to Zigbee2mqtt Core ...
> zigbee2mqtt@1.25.1 start
> node index.js
Zigbee2MQTT:info  2022-05-10 23:10:14: Logging to console and directory: '/config/zigbee2mqtt/log/2022-05-10.23-10-12' filename: log.txt
Zigbee2MQTT:info  2022-05-10 23:10:14: Starting Zigbee2MQTT version 1.25.1 (commit #unknown)
Zigbee2MQTT:info  2022-05-10 23:10:14: Starting zigbee-herdsman (0.14.27)
Zigbee2MQTT:info  2022-05-10 23:10:15: zigbee-herdsman started (resumed)
Zigbee2MQTT:info  2022-05-10 23:10:15: Coordinator firmware version: '{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20190608,"transportrev":2},"type":"zStack12"}'
Zigbee2MQTT:info  2022-05-10 23:10:15: Currently 0 devices are joined:
Zigbee2MQTT:warn  2022-05-10 23:10:15: `permit_join` set to  `true` in configuration.yaml.
Zigbee2MQTT:warn  2022-05-10 23:10:15: Allowing new devices to join.
Zigbee2MQTT:warn  2022-05-10 23:10:15: Set `permit_join` to `false` once you joined all devices.
Zigbee2MQTT:info  2022-05-10 23:10:15: Zigbee: allowing new devices to join.
Zigbee2MQTT:info  2022-05-10 23:10:15: Connecting to MQTT server at mqtt:192.168.178.18
Zigbee2MQTT:info  2022-05-10 23:10:15: Connected to MQTT server
Zigbee2MQTT:info  2022-05-10 23:10:15: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'
Zigbee2MQTT:info  2022-05-10 23:10:15: Started frontend on port 0.0.0.0:8099
Zigbee2MQTT:info  2022-05-10 23:10:15: MQTT publish: topic 'zigbee2mqtt/bridge/config', payload '{"commit":"unknown","coordinator":{"meta":{"maintrel":3,"majorrel":2,"minorrel":6,"product":0,"revision":20190608,"transportrev":2},"type":"zStack12"},"log_level":"info","network":{"channel":11,"extendedPanID":"0xdddddddddddddddd","panID":6754},"permit_join":true,"version":"1.25.1"}'
Zigbee2MQTT:info  2022-05-10 23:10:15: MQTT publish: topic 'zigbee2mqtt/bridge/state', payload 'online'`

I'm using Mosquito Broker on Port 1833, i think its working, but i can't add new devices.
Can somebody help me again?
Stefan
I tested with two devices

**Comment by Stefan-Mav** [comment]
Now its working,
The path from the server was not correct.

**Comment by Stefan-Mav** [comment]
one day later, a lot of working with reset on devices everything is working again and the Insta is working to.
Thanks for the help.
Stefan
